htdocs/core/login/functions_openid_connect.php

```diff
@@ -42,12 +42,6 @@ function check_user_password_openid_connect($usertotest, $passwordtotest, $entit
 {
 	global $db;
 
-	if (getDolGlobalInt('MAIN_MODULE_OPENIDCONNECT', 0) <= 0) {
```

Comment:
I have pushed a change into develop to introduce an option MAIN_AUTHENTICATION_OIDC_ON so we can enable/disable OIDC globaly from setup.
Can you replace MAIN_MODULE_OPENIDCONNECT with MAIN_AUTHENTICATION_OIDC_ON instead of removing ?

Reply:
@eldy no :-( this is my first entry into openid, and I run my own authentik where I think I still have configuration issues.

It might not help that I run both my Dolibarr and my Authentik inside separate podman pods, only exposing the ports that should be exposed - and further more both run on my mac laptop in podman desktop.